core/src/apps/bitcoin/sign_tx/layout.py

```diff
@@ -310,14 +310,17 @@ async def confirm_nondefault_locktime(lock_time: int, lock_time_disabled: bool)
         if lock_time < _LOCKTIME_TIMESTAMP_MIN_VALUE:
             text = TR.bitcoin__locktime_set_to_blockheight
             value = str(lock_time)
+            value_text_mono = True
         else:
             text = TR.bitcoin__locktime_set_to
             value = format_timestamp(lock_time)
+            value_text_mono = False
         await layouts.confirm_value(
             TR.bitcoin__confirm_locktime,
             value,
             text,
             "nondefault_locktime",
+            value_text_mono=value_text_mono,
```

Comment:
right. while we're at this, plesae rename "value text _mono_" to something semantic, such as `is_blob`
("mono" isn't even the right thing here, what we care about is the linebreak strategy and not necessarily the font)

Reply:
Like this? https://github.com/trezor/trezor-firmware/pull/4372/commits/d7889cd2d9defdd6829e5be6b51b96920947441a